app/models/rental.py

```diff
@@ -0,0 +1,12 @@
+from app import db
+
+class Rental(db.Model):
+    __tablename__ = "rentals"
+    rental_id = db.Column(db.Integer, primary_key=True)
+    due_date = db.Column(db.DateTime, nullable=True)
+    customer_id = db.Column(db.Integer, db.ForeignKey("customers.customer_id"))
+    video_id = db.Column(db.Integer, db.ForeignKey("videos.video_id"))
+    customers = db.relationship("Customer", backref="rentals")
+    videos = db.relationship("Video", backref="rentals")
+
```

Comment:
You may want to add a check in date to know if the user returned this rental:

```suggestion
    check_in_date = db.Column(db.DateTime, nullable=True)
```